pages/data/dataset_info/_id.vue

```diff
@@ -135,8 +142,10 @@ export default {
       requestUrl: null,
       stations: [],
       title: null,
-      uriDatasetDef: null,
-      wafDataset: null,
+      // uriDatasetDef: null,
+      // wafDataset: null,
+      wafURL: [],
+      doublewaf: null,
```

Comment:
```suggestion
```

Rename and move this into a `computed` variable.

So within the `computed: { }`, add:

```js
hasSecondWAF: function() {
  return this.wafURL.length > 1
}
```